spring-cloud-contract-verifier/src/main/groovy/org/springframework/cloud/contract/verifier/builder/JaxRsBodyParser.java

```diff
@@ -22,15 +22,19 @@
 	};
 
 	default String readEntity() {
-		return "response.readEntity(String.class)";
+		String classToCastTo = "String"
```

Comment:
Here also rather than having conditions, it might be better to have `KotlinJaxRsBodyParser`, `JavaJaxRsBodyParser`, etc. ; @marcingrzejszczak wdyt?